table.md

```diff
@@ -151,67 +151,67 @@
 | Ingo, stop doing this kind of crap. ... You seem to be in total denial. ... Stop it. That kind of "head-in-the-sand" behavior is not conducive to good code, ... Seriously. If you can't make the non-THP case go faster, don't even bother sending out the patches. Similarly, if you cannot take performance results from others, don't even bother sending out the patches. ... So stop ignoring the feedback, and stop shooting the messenger. Look at the numbers, and admit that there is something that needs to be fixed. | 0.839837190744 |
 | This is disgusting. Many (most?) __gup_fast() users just want a single page, and the stupid overhead of the multi-page version is already unnecessary. This just makes things much worse. | 0.84016427745 |
 | So I absolutely *hate* how this was done.... I'm pulling it this time, but quite frankly, next time I see this kind of ugly AND TOTALLY POINTLESS layering violation, I will just drop the stupid pull request. ... In other words, this was NOT OK. This was stupid and wrong, and violated all sanity. ... What the hell was going on here? | 0.840723199203 |
 | This has so much wrong that I don't know where to start. | 0.841706410183 |
-| NO I AM NOT! Dammit, this feature is f*cking brain-damaged. ...But even apart from the Xen case, it was just a confusing hell. Like Yoda said: "Either they are the same or they are not. There is no 'try'". So pick one solution. Don't try to pick the mixed-up half-way case that is a disaster and makes no sense. | 0.842414855313 |
-| So I have to say, I hate this entire series. ... So quite frankly, I think the whole series is total and utter garbage. And there isn't even any *explanation" for the garbage. You say that you are unifying things, but dammit, in order to unify them you end up *adding*new*f&^#ing*code*. ... So a honking big NAK on this whole series unless you can explain with numbers and show with numbers what the advantage of the abortion is. | 0.842510012478 |
+| NO I AM NOT! Dammit, this feature is f\*cking brain-damaged. ...But even apart from the Xen case, it was just a confusing hell. Like Yoda said: "Either they are the same or they are not. There is no 'try'". So pick one solution. Don't try to pick the mixed-up half-way case that is a disaster and makes no sense. | 0.842414855313 |
+| So I have to say, I hate this entire series. ... So quite frankly, I think the whole series is total and utter garbage. And there isn't even any *explanation" for the garbage. You say that you are unifying things, but dammit, in order to unify them you end up *adding\*new\*f&^#ing\*code*. ... So a honking big NAK on this whole series unless you can explain with numbers and show with numbers what the advantage of the abortion is. | 0.842510012478 |
```

Comment:
Seems like in this instance the \* are not for censorship, should we also escape them?

Reply:
Nice catch! I should probably have made the replacement regex stricter, so that it only escapes `*` if surrounded by `[Ff]` and `[cC][kK]`, or `[sS][hH]` and `[tT]`.

Although that will still result in this being rendered weirdly. 

With escapes
>*adding\*new\*f&^#ing\*code*

Without escapes
>*adding*new*f&^#ing*code*

Perhaps *all* `*`s should be escaped? I personally like the rendered italics, but I'm pretty sure Linus wasn't intentionally writing Markdown syntax. In this case, it looks like Linus intended to put *a lot* of emphasis on each word. Escaping all `*`s might better represent how Linus *intended* his rants to be viewed/read. But the other option might be to customize how this is found/replaced, so it becomes something like
>***adding new f&^#ing code***

Reply:
> Escaping all *s might better represent how Linus intended his rants to be viewed/read.

I'm also leaning towards this as the best way to preserve the meaning behind his words, as it is how we intended them to be viewed.

Reply:
What do you think about switching to another formatting language other than markdown?

Reply:
I'm not really familiar with [the formatting languages that GitHub supports](https://github.com/github/markup/blob/master/README.md#markups) besides Markdown. Some of those supported languages look like they'll run into the same problem of using `*` for emphasis.

But another option might be to deploy with GitHub pages. [GitHub provides a tutorial for hosting HTML here](https://pages.github.com/). Or we could do [something more fancy](https://cli.vuejs.org/guide/deployment.html#github-pages).

Reply:
I made a [basic deployment](https://spenserblack.github.io/linusrants/) as an example. It definitely needs some work, but it's the basic idea.